Packs/Zscaler/Integrations/Zscaler/Zscaler_test.py

```diff
@@ -180,3 +180,21 @@ def test_url_multiple_arg(url, multiple, expected_data):
     request_data = m.last_request.json()
     assert len(request_data) == len(expected_data)
     assert request_data == expected_data
+
+
```

Comment:
please consider inserting a test case for multiple ip submission in 1/2 of the commands.